docs/app/Views/Layouts.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { Header, Segment, Container, Divider, Card } from 'semantic-ui-react'
+
+const LayoutsExample = () => (
+  <Container as='div'>
```

Comment:
Heads up, the default component class is `div` so we can remove all `as='div'` props since it is redundant.